src/domain/insert_email.py

```diff
@@ -13,21 +13,23 @@
 from PySide6.QtGui import QIcon
 from PySide6.QtWidgets import QDialog, QDialogButtonBox
 
+from domain.email_notifier import EmailNotifier
+from domain.notifier import Notifier, notifiers
 from ui.ui_insert_email import Ui_DialogInsertEmail
 
 
 class DialogInsertEmail(QDialog, Ui_DialogInsertEmail):
     """Class to insert email configuration to enable WADAS for email notifications."""
 
-    def __init__(self, email_configuration):
+    def __init__(self):
         super(DialogInsertEmail, self).__init__()
         self.ui = Ui_DialogInsertEmail()
         self.ui.setupUi(self)
         self.setWindowIcon(
             QIcon(os.path.join(os.getcwd(), "img", "mainwindow_icon.jpg"))
```

Comment:
Relying on os.getcwd() might cause problems if the current directory is changed.
For static resources maybe it's better to retrieve the path of the current module at the beginning and setting it as a global variable, so that it can be used everywhere (and efficiently) inside the module without issues.